src/models/Website.php

```diff
@@ -102,7 +107,33 @@ public function ngRestAttributeTypes()
                 'query' => $this->getLang(),
                 'relation' => 'lang',
                 'labelField' => ['name']
-            ]
+            ],
+            'group_ids' => [
+                'class' => CheckboxList::class,
+                'data' => array_merge(
+                    [0 => Module::t('model_website_all')],
+                    Group::find()
```

Comment:
Can you please verify if this is loaded also in frontend context? If so we have to change that, as those data should only load in admin context. Otherwise this would be a very bad changed regarding load time and database requests